recipes-debian/bash/files/ignore-test-result.patch

```diff
@@ -0,0 +1,18 @@
+--- bash-5.0/tests/run-all	2019-07-12 11:18:17.302223153 +0900
++++ bash-5.0.new/tests/run-all	2019-07-12 11:30:49.676925222 +0900
+@@ -42,8 +42,13 @@
+ 	*)	echo $x
+ 		output=`sh $x`
+ 		if [ -n "$output" ]; then
+-			echo "$output"
+-			echo "FAIL: $x"
++		        echo "$output"
```

Comment:
This line should be same as L7, but SPACEs are used instead of a TAB.
Could you fix it and other places?

Reply:
thanks. I've fixed that and made v2 of pull request #13